crates/rpc/src/handlers/block.rs

```diff
@@ -86,28 +80,54 @@ pub async fn get_beacon_block_from_id(
         )))
 }
 
-/// Called by `/eth/v2/beacon/{block_id}/attestations` to get block attestations
-pub async fn get_block_attestations(block_id: ID, db: ReamDB) -> Result<impl Reply, Rejection> {
-    let beacon_block = get_beacon_block_from_id(block_id, &db).await?;
+/// Called by `/genesis` to get the Genesis Config of Beacon Chain.
+#[get("/beacon/genesis")]
+pub async fn get_genesis(db: web::Data<ReamDB>) -> actix_web::Result<impl Responder> {
+    Ok(HttpResponse::Ok())
+}
+
+/// Called by `/eth/v2/beacon/blocks/{block_id}/attestations` to get block attestations
+#[get("/beacon/blocks/{block_id}/attestations")]
+pub async fn get_block_attestations(
+    db: web::Data<ReamDB>,
+    block_id: web::Path<String>,
```

Comment:
```suggestion
    block_id: web::Path<ID>,
```

We don't want to use strings, shouldn't actix handle this?